index-import.cpp

```diff
@@ -280,77 +360,32 @@ int main(int argc, char **argv) {
     return EXIT_FAILURE;
   }
 
-  bool success = true;
-
-  for (auto &InputIndexPath : InputIndexPaths) {
-    InputIndexPath = normalizePath(InputIndexPath);
+  if (ParallelStride == 0 || ParallelStride >= InputIndexPaths.size()) {
```

Comment:
I have a thought here, and I'm not sure how I feel about it, but it's worth mentioning. The non-parallel case could still go down the `dispatch_apply` code path, with `stride =  InputIndexPaths.size ()`. On one hand, it reduces the little bit of duplication, but on the other hand it would make the non-parallel case less clear.

Reply:
Hmm, not sure, your call. My initial reaction is to leave it as-is as there could be subtle differences between the two, but now there are two separate code paths to test.

Reply:
My instinct is to leave it as is too.